src/AvtaleOversikt/Filtrering/DeltakerOgBedriftFilter.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { FormEvent, FunctionComponent, useContext, useState } from 'react';
+import { FiltreringProps } from '@/AvtaleOversikt/Filtrering/Filtrering';
+import { Filter } from '@/AvtaleOversikt/Filtrering/Filter';
+import { validerFnr } from '@/utils/fnrUtils';
+import { SøkeInput } from '@/AvtaleOversikt/Filtrering/SøkeInput';
+import { SkjemaelementFeil } from 'nav-frontend-skjema/lib/skjemaelement-feilmelding';
+import { validerOrgnr } from '@/utils/orgnrUtils';
+import { InnloggetBrukerContext } from '@/InnloggingBoundary/InnloggingBoundary';
+import { Radio } from 'nav-frontend-skjema';
+
+type Validering = (verdi: string) => SkjemaelementFeil | undefined;
+
+const fnrValidering: Validering = verdi => (!validerFnr(verdi) ? { feilmelding: 'Ugyldig fødselsnummer' } : undefined);
+const orgNrValidering: Validering = verdi =>
+    !validerOrgnr(verdi) ? { feilmelding: 'Ugyldig bedriftsnummer' } : undefined;
+
+type Søketype = 'deltaker' | 'bedrift' | 'egne';
+
+export const DeltakerOgBedriftFilter: FunctionComponent<FiltreringProps> = props => {
+    const [søketype, setAktueltSøk] = useState<Søketype>('egne');
```

Comment:
Kan vi kalle denne for aktivSøketype, setAktivSøkeyupe ?

Reply:
Renamet